moveit_core/collision_detection/src/pycollision_detection.cpp

```diff
@@ -0,0 +1,67 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/eigen.h>
+#include <moveit/collision_detection/world.h>
+#include <moveit/collision_detection/collision_matrix.h>
+#include <moveit/collision_detection/collision_common.h>
+#include <moveit/python/pybind_rosmsg_typecasters.h>
+
+namespace py = pybind11;
+
+using namespace collision_detection;
+
+auto SET_ENTRY1 = py::overload_cast<const std::string&, const std::string&, bool>(&AllowedCollisionMatrix::setEntry);
```

Comment:
I prefer inlining this definition where required. It's only used once, isn't it?

Reply:
I did this because there are many more overloads that presumably we should implement, I just didn't do all of them right now.

Reply:
Also one thing I struggled a lot with is formatting these overload definitions, they're really long so putting them up here is actually nicer for formatting.